shard-operator/src/test/java/com/redhat/service/bridge/shard/operator/controllers/BridgeIngressControllerTest.java

```diff
@@ -76,7 +76,7 @@ private BridgeIngress buildBridgeIngress() {
         BridgeIngress bridgeIngress = new BridgeIngress();
         bridgeIngress.setMetadata(
                 new ObjectMetaBuilder()
-                        .withName(KubernetesResourceUtil.sanitizeName(TestConstants.BRIDGE_ID))
+                        .withName(BridgeIngress.buildResourceName(TestConstants.BRIDGE_ID))
```

Comment:
I think we might consider a builder for the `BridgeIngress`, turning its constructor private since there are some caveats on creating the object. We already have the `fromDTO`, but a fluent/builder API should fit perfectly, and then we could align the creation of the tests better. Additionally, in the future, if we would need to create this object without using a DTO, it would come in handy.

Reply:
Hi @ricardozanini , would you take over this ticket? I'm trying to figure out why the CI on https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/126 is failing randomly and I do not have much time to follow up on this

Reply:
Of course! Do you say this PR here? We can merge as it is and open a follow-up one for this minor refactoring. wdyt?

Reply:
Yes fine for me!

Reply:
@sutaakar fine for you?

Reply:
Fine for me too.